depthai_ros_driver/CMakeLists.txt

```diff
@@ -58,6 +59,7 @@ include_directories (include)
 include_directories (SYSTEM
     ${Boost_INCLUDE_DIRS}
     ${OpenCV_INCLUDE_DIRS}
+    ${MSGPACK}/include
```

Comment:
We should configure msgpack to use C++17: https://github.com/msgpack/msgpack-c/wiki/v2_0_cpp_configure#msgpack_cxx17-c-only